packages/runner/src/execution/slack-ask-question-handler.ts

```
@@ -0,0 +1,21 @@
+  try {
+    const result = await step.waitForReply(timeoutMs);
+    const exchange = {
+      question: step.input.question,
+      timestamp: new Date(),
+      reply: result.reply,
+      timeout: result.timeout === true
+    };
+    await updateRunAskQuestionExchanges(runId, exchange);
+  } catch (error) {
+    throw error;
+  }
```

ESLint

[error] Unnecessary try/catch wrapper.

(no-useless-catch)